modules/apps/dynamic-data-mapping/dynamic-data-mapping-taglib/src/main/resources/META-INF/resources/html/start.jsp

```diff
@@ -136,6 +136,16 @@
 
 			var ddmFormDefinition = <%= DDMUtil.getDDMFormJSONString(ddmForm) %>;
 
+			<%
+			if (defaultLocale != null) {
+			%>
+
```

Comment:
@laszlopap, this blank line can be the issue, please remove it and re-run SF check to see how it goes. Try `gradlew formatSource` on module `dynamic-data-mapping-taglib` or `ant format-source-current-branch` on `portal-impl`. 🙂